docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,152 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Table functions
+---------------
+
+The connector provides the following table functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries to expose row-level changes
+between two versions of a Delta Lake table.
+When the ``change_data_feed_enabled`` table property is set to ``true``
+on a specific Delta Lake table, the connector records change events for
+all data changes on the table. In addition to columns present in the table
+the function returns the following values for each change event:
+
+* ``_change_type``
+    Gives the type of change that occurred as represented as ``VARCHAR`` value.
+    Possible values are ``insert``, ``delete``, ``update_preimage`` and ``update_postimage``.
+
+* ``_commit_version``
+    Shows the table version for which the change occurred as ``BIGINT`` value.
+
+* ``_commit_timestamp``
+    Represents the timestamp for the commit in which the specified change happened as
+    ``TIMESTAMP(3) WITH TIME ZONE`` data type value.
+
+.. code-block:: sql
+
+    SELECT
+      *
+    FROM
+      TABLE(
+        system.table_changes(
+          schema_name => 'test_schema',
+          table_name => 'tableName',
+          since_version => 0
+        )
+      );
+
+``schema_name`` - type ``VARCHAR``, required, name of the schema for which the function will be called
+
+``table_name`` - type ``VARCHAR``, required, name of the table for which the function will be called
+
+``since_version`` - type ``BIGINT``, optional, version from which changes will be shown, exclusive
```

Comment:
```suggestion
``since_version`` - type ``BIGINT``, optional, version from which changes are shown, exclusive
```

Also what do ou mean by exclusive? If you mean that it work show that version itself we should maybe reword